config/config.go

```diff
@@ -520,6 +520,8 @@ type TiKVClient struct {
 	GrpcKeepAliveTimeout uint `toml:"grpc-keepalive-timeout" json:"grpc-keepalive-timeout"`
 	// GrpcCompressionType is the compression type for gRPC channel: none or gzip.
 	GrpcCompressionType string `toml:"grpc-compression-type" json:"grpc-compression-type"`
+	// PDServerTimeout is the max time which PD client will wait for the PD server.
+	PDServerTimeout uint `toml:"pd-server-timeout" json:"pd-server-timeout"`
```

Comment:
Why put it in `TiKVConfig`? Is it the timeout between TiDB and PD or between TiKV and PD?

Reply:
Updated.